web/packages/design/src/DataTable/Pager/StyledPager.tsx

```diff
@@ -39,7 +39,7 @@ export const StyledArrowBtn = styled.button`
   &:disabled {
     cursor: default;
     ${Icon} {
-      opacity: 0.1;
+      opacity: 0.2;
```

Comment:
Oh, this must have been missed. I'm inspecting the active arrow, and it is only .5 opacity; we should have a minimum "muted" white at .54 opacity, and disabled at .36 opacity. I would go with the "slightly muted" setting of .72 for the active icon and .36 for the disabled one...

But now that I'm thinking about it, are these custom-implemented icon buttons? Is there a reason these aren't leveraging our icon button component?

Reply:
This is definitely a good place, it looks like we are missing those colors. I can add them.

Reply:
Ah yes I had missed the new text opacities because I hadn't noticed that they had changed, sorry about that. We should update them. I notice though that in the theme design guidelines we don't have a `placeholder` text opacity, which we use in all of our inputs, could you add one please @roraback?

Reply:
I reverted this change in the PR, I'll add it in another PR as I found one other thing that we can improve in the table.